.github/workflows/build.yml

```diff
@@ -0,0 +1,58 @@
+name: CI
+
+on:
+  workflow_dispatch:
+    inputs:
+      keep_workdir:
+        required: false
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  check:
+    if: |
+      github.event.issue.pull_request &&
+      startsWith(github.event.comment.body, '/ost') &&
```

Comment:
We probably want to use a different verb than `/ost`

Reply:
eventually sure